examples/compass.py

```diff
@@ -1,7 +1,16 @@
+"""
+    compass.py
+    ~~~~~~~~~~
+
+    Creates a compass.
+
+    To calibrate the compass, lie it flat and spin it horizontally. For best
```

Comment:
http://blog.oxforddictionaries.com/2012/03/lie-or-lay/ suggests "lay it
flat" is correct. I'd probably say "lie it flat". I suspect an 11 year old
might too.

Michael.

On 30 October 2015 at 15:30, Bryan Morrissey <notifications@github.com>
wrote:

> In examples/compass.py
> <https://github.com/bbcmicrobit/micropython/pull/60#discussion_r43515101>:
>
> > @@ -1,7 +1,16 @@
> > +"""
> > +    compass.py
> > +    ~~~~~~~~~~
> > +
> > +    Creates a compass.
> > +
> > +    To calibrate the compass, lie it flat and spin it horizontally. For best
>
> Minor grammatical question - should this be "lie it flat" or "lay it
> flat"? The latter sounds better to my ears, but I can't say I'm certain
> which is correct.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bbcmicrobit/micropython/pull/60/files#r43515101>.
>